tachyon/math/elliptic_curves/msm/algorithms/cuzk/cuzk_unittest.cc

```diff
@@ -72,13 +73,11 @@ TEST_F(CUZKTest, RunWithRandom) {
 
   auto bases = gpu::GpuMemory<bn254::G1AffinePointGpu>::MallocManaged(size);
   for (size_t i = 0; i < bases.size(); ++i) {
-    bases[i] = bn254::G1AffinePointGpu::FromMontgomery(
-        test_set.bases[i].ToMontgomery());
+    bases[i] = ConvertPoint<bn254::G1AffinePointGpu>(test_set.bases[i]);
   }
   auto scalars = gpu::GpuMemory<bn254::FrGpu>::MallocManaged(size);
   for (size_t i = 0; i < scalars.size(); ++i) {
-    scalars[i] =
-        bn254::FrGpu::FromMontgomery(test_set.scalars[i].ToMontgomery());
```

Comment:
Can this be added to the first commit?

Reply:
I think it's possible. But isn't it more likely removing montgomery conversions?

Reply:
That's fair. I suppose this isn't an explicit cast but more of a conversion.